app/src/main/java/com/stevesoltys/seedvault/settings/SettingsManager.kt

```diff
@@ -143,6 +143,15 @@ class SettingsManager(private val context: Context) {
 
     fun isBackupEnabled(packageName: String) = !blacklistedApps.contains(packageName)
 
+    /**
+     * Disables backup for an app. Similar to [onAppBackupStatusChanged].
+     */
+    fun disableBackup(packageName: String) {
+        if (blacklistedApps.add(packageName)) {
```

Comment:
pedantic: can call it a denylist, that's the term used everywhere in AOSP now.

Reply:
yep, but maybe in a separate MR. There's so many open MRs already and didn't want to do unnecessary changes here.